### ISSUE TITLE
Question: base_score different values

### ISSUE THREAD

**Issue by fakyras** [dmlc/xgboost#1672]
I did not find if anyone came up with setting different base_scores for each training row for first xgboost iteration? In a case i am working on it could speed up xgboost training time drastically; It would also be a nice option as a "continued" training, using initial biases from other pre-trained model. Would this be hard to implement?

**Comment by wangchao95** [comment]
Agreed. In my case which is a regression problem, I tried a series of different initial values(i.e. mean of the label) and of course gained different results and I think using pre-trained/first-guess predictions as initial values would be very useful. Found a similar issue which came from a multi-classification problem #1380

A single initial value could be enough in some cases. But in other cases, if we can have a much better initial sequence to start with, the result could reach a better local-optimum. Hope to raise attention.

**Comment by khotilov** [comment]
See #2222
